src/lib.rs

```diff
@@ -281,6 +281,17 @@ impl Url {
     }
 
     /// Return a default `ParseOptions` that can fully configure the URL parser.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use url::Url;
+    ///
+    /// let parse_options = Url::options();
+    /// let base = parse_options.parse("https://example.net/api/").unwrap();
```

Comment:
Please come up with an example that requires ParseOptions. This one would just use `Url::parse`.